src/core/components/SiteNotice/styles.scss

```diff
@@ -0,0 +1,3 @@
+.SiteNotice .Notice-column {
+  justify-content: center;
```

Comment:
Since it seems like a bug that this will look different from [other notices](https://design.firefox.com/photon/components/message-bars.html), maybe add a comment explaining that it improves the look of full-width messages on wide screens?